coalib/bearlib/__init__.py

```diff
@@ -119,7 +127,11 @@ def wrapping_function(*args, **kwargs):
                     else:
                         kwargs[arg] = depr_arg_value
                     del kwargs[deprecated_arg]
-            return func(*args, **kwargs)
+            profile_bears = kwargs.get('profile_bears', False)
```

Comment:
E714 test for object identity should be 'is not'

*Origin: PycodestyleBear (E714), Section: `autopep8`.*